mail/helm-install-prerequisites.adoc

```diff
@@ -0,0 +1,22 @@
+=== Install OpenTelemetry Collector on OpenShift
```

Comment:
This title is not correct, it should be "Install the Mail server on Openshift" or something like that